frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -202,24 +210,38 @@ export class Wizard {
     const { name, description, template, provisionSource, os, flavor, workload } = data;
     if (name) {
       await this.fillName(name);
+      // eslint-disable-next-line no-console
+      console.log('VM name updated: ', name);
```

Comment:
please do nto commit the console logs

Reply:
Doublechecked that file, I don't have `console.log` there